### PR TITLE
Require and document use of time-sync

### PR DESCRIPTION
**Pull request by kalikiana**
In packaging terms ntp-daemon is the counterpart to the time-sync target required by systemd units.

See: https://progress.opensuse.org/issues/125372

### CODE REVIEW

File: dependencies.yaml [hunk 1]
```diff
@@ -160,6 +160,7 @@ worker_requires:
   optipng:
   sqlite3: '>= 3.24.0'   # for INSERT INTO … ON CONFLICT … DO UPDATE SET … required by cache service
   bsdtar:  # for extraction feature of OpenQA::Downloader
+  ntp-daemon:
```
Comment (Martchus) [inline dependencies.yaml:163]: Souldn't this be a common dependency? Syncing just the worker's clock won't help if the web UI is still out of sync.

**Review by Martchus — CHANGES_REQUESTED** [review]
You also need to run the script for updating the actual spec file.

**Review by perlpunk — APPROVED** [review]

File: dist/rpm/openQA.spec [hunk 2]
```diff
@@ -56,7 +56,7 @@
 # The following line is generated from dependencies.yaml
 %define client_requires curl git-core jq perl(Getopt::Long::Descriptive) perl(IO::Socket::SSL) >= 2.009 perl(IPC::Run) perl(JSON::Validator) perl(LWP::Protocol::https) perl(LWP::UserAgent) perl(Test::More) perl(YAML::PP) >= 0.020 perl(YAML::XS)
 # The following line is generated from dependencies.yaml
-%define worker_requires bsdtar openQA-client optipng os-autoinst < 5 perl(Capture::Tiny) perl(File::Map) perl(Minion::Backend::SQLite) >= 5.0.7 perl(Mojo::IOLoop::ReadWriteProcess) >= 0.26 perl(Mojo::SQLite) psmisc sqlite3 >= 3.24.0
+%define worker_requires bsdtar ntp-daemon openQA-client optipng os-autoinst < 5 perl(Capture::Tiny) perl(File::Map) perl(Minion::Backend::SQLite) >= 5.0.7 perl(Mojo::IOLoop::ReadWriteProcess) >= 0.26 perl(Mojo::SQLite) psmisc sqlite3 >= 3.24.0
```
Comment (perlpunk) [inline dist/rpm/openQA.spec:59]: I think that is still wrong, it's in `worker_requires` not `common_requires`

Comment (kalikiana) [reply dist/rpm/openQA.spec:59]: I think I ran update-deps before editing dependencies.yaml 🤦🏾 😅

**Review by mimi1vx — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Comment by okurz** [comment]
This triggered "unresolvable"

```
unresolvable: have choice for ntp-daemon: chrony ntp
```

fixed in https://build.opensuse.org/projects/devel:openQA/prjconf with
```
Prefer: … chrony
```

might need the same adjustement in more projects

**Pull request merged**